### DISCUSSION TITLE
visual studio code

### DISCUSSION THREAD

**Discussion by usha-ui** [git-for-windows/git#3003] [General]
by mistake i have deleted git folder from visual studio code , and when i am using sourse control it is showing ERROR no sourse control provider registered. 

- what to do?
- completely uninstall visual studio code OR
- how to fix the problem.

**Comment by dscho** [comment]
The `.git` folder contains all the information Git requires. Deleting it is unfortunately a fatal, non-recoverable problem. There is nothing you can do to fix this, that information is gone.
